packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -91,11 +91,23 @@ class CollapsibleMenuView extends React.Component {
   render() {
     const { children, boundingRef, menuWidth, ...customProps } = this.props;
     const visibleChildren = React.Children.toArray(children);
+
     let hiddenChildren = null;
+    let singleHiddenIconChild = false;
     if (this.state.hiddenStartIndex >= 0) {
       hiddenChildren = visibleChildren.splice(this.state.hiddenStartIndex);
+      singleHiddenIconChild  = hiddenChildren.length === 1 && hiddenChildren[0].props.isIconOnly;
```

Comment:
look like have an two spaces before the `=` here

Reply:
Dang. Knew I shouldn't have updated via the browser 😆